common/changes/@itwin/appui-react/stagepanel-pinned-fix_2023-04-05-15-17.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "Fix bug where pinned stage panel def prop is out of sync with its react element. Fix layout info widget not updating info on change stage panel def change.",
```

Comment:
From these comments a changelog file is generated. Reword first sentence and instead of mentioning test-app specific layout info you could mention that an event is added to listen to pinned changes. You could also add multiple changes objects instead of adding everything into a single comment property.